crates/sui-core/src/authority.rs

```diff
@@ -2428,6 +2428,30 @@ impl AuthorityState {
             .compute_object_reference())
     }
 
+    pub fn get_authenticator_state_start_version(&self) -> SuiResult<Option<SequenceNumber>> {
+        // should not be called if we have not reached the protocol version where this object
+        // exists
+        if self
+            .load_epoch_store_one_call_per_task()
```

Comment:
Not a huge deal but could be safer to take the epoch store as argument here in this function.

Reply:
this function is now removed